internal/gorson/io/io.go

```diff
@@ -68,7 +70,13 @@ func ReadFromParameterStore(path util.ParameterStorePath) map[string]string {
 	return values
 }
 
-func writeSingleParameter(c chan string, client *ssm.SSM, name string, value string, retryCount int) {
+// WriteResult is the result writing a single parameter - successful is Error is nil
+type WriteResult struct {
+	Name  string
+	Error error
+}
+
+func writeSingleParameter(c chan WriteResult, client ssmiface.SSMAPI, name string, value string, retryCount int) {
```

Comment:
I updated this function signature to return a "write result", rather than just a string, so the caller can keep track of errors